rtmbot.py

```diff
@@ -55,12 +55,19 @@ def output(self):
         for plugin in self.bot_plugins:
             limiter = False
             for output in plugin.do_output():
-                channel = self.slack_client.server.channels.find(output[0])
-                if channel != None and output[1] != None:
+                destination = output[0]
+                message = output[1]
+                # things that start with U are users. convert to an IM channel.
+                if destination.startswith('U'):
```

Comment:
Mind also checking for `'G'` (which is a [Group Message](https://api.slack.com/methods/groups.open))?

```
elif destination.startswith('G'):
    result = self.slack_client.api_call('groups.open'), channel=destination)
#...
```